deeds-dapp-webapp/src/main/webapp/vue-app/main.js

```diff
@@ -509,6 +509,20 @@ const store = new Vuex.Store({
             state.provider
           );
         }
+        if (state.sushiswapPairAddress) {
+          state.sushiswapPairContract = new ethers.Contract(
```

Comment:
you have already an `lpContract` defined for all supported LP Tokens inside `LiquidityPool.vue` which is generic and suffiecient to retrieve all information.